### PR TITLE
PySide6 and Maya 2025 fixes, UE 5.4 turntable project

### PR DESCRIPTION
**Pull request by gplsteph**
Fixed some problems for PySide6 and Maya 2025 which does not ship `six` with its Python interpreter anymore.
Added a turntable projects for UE 5.2, 5.3 and 5.4, generated from existing projects.
Fixed problems from latest tk-multi-publish2.
Ensured temporary paths on Windows do not include the user name which could cause UE to crash.

### CODE REVIEW

File: hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py [hunk 1]
```diff
@@ -970,15 +980,22 @@ def publish(self, settings, item):
         publish_path = self.get_publish_path(settings, item)
         publish_path = os.path.normpath(publish_path)
 
-        # This plugin publishes a turntable movie to Shotgun
+        # This plugin publishes a turntable movie to SG
```
Comment (carlos-villavicencio-adsk) [inline hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py]:
```suggestion
        # This plugin publishes a turntable movie to FPTR
```

File: hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py [hunk 2]
```diff
@@ -970,15 +980,22 @@ def publish(self, settings, item):
...
         # Another folder can be specified as long as the name has no spaces
         # Spaces are not allowed in command line Unreal Python args
-        temp_folder = tempfile.mkdtemp(suffix="temp_unreal_shotgun")
+
+        # Set a base temp dir on Windows to avoid having
+        # the user name in the temp path which can include
+        # "." e.g. firstname.name and makes UE crashes
+        base_temp_dir = None
+        if sys.platform == "win32":
+            base_temp_dir = r"C:\Temp"
```
Comment (carlos-villavicencio-adsk) [inline hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py:997]: Is it safe to assume this directory exists on the filesystem?

Comment (gplsteph) [reply hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py:997]: Not 100% sure actually. But I didn't create this directory myself, and it is referenced in [this](https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-delete-temporary-files-in-Windows.html)

Comment (carlos-villavicencio-adsk) [reply hooks/tk-multi-publish2/tk-maya/basic/publish_turntable.py:997]: Sounds fair.

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

**Pull request merged**